examples/simple-platform/platform-structure.md

```diff
@@ -0,0 +1,45 @@
+# Platform structure
+
+Our platform has two server nodes. One runs the `platform-deploy` hypervisor 
+and the other runs the `web-auth-endpoint` hypervisor. We also have two 
+browser nodes - one for the accounts app and another for the template app. 
```

Comment:
I feel like this sentence contains an implied inversion of control. We have two apps which load n browser nodes - the nodes do not load the apps. The browser nodes are transient and only run for as long as a user has the app open in their browser though this sentence implies the nodes are always there.